src/Tribe/Admin/Notice/Install_Event_Tickets.php

```diff
@@ -0,0 +1,438 @@
+			$html,
+			[
+				'dismiss' => true,
+				'type'    => 'warning',
+			]
+		);
+	}
+
+	/**
+	 * Enqueue assets required for the notice.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	private function enqueue_assets() {
+		wp_enqueue_style( 'wp-components' );
+		tribe_asset_enqueue_group( self::$assets_group );
+	}
+
+	/**
+	 * Activate notice for `Event Tickets`.
+	 *
+	 * @since TBD
+	 */
+	public function notice_activate() {
+		if ( ! $this->should_display_notice_activate() ) {
+			return '';
+		}
+
+		$this->enqueue_assets();
+
+		$args = [
+			'description'  => __( 'You\'re almost there! Activate Event Tickets to manage attendee registration and ticket sales to your events, for free.', 'the-events-calendar' ),
+			'button_label' => __( 'Activate Event Tickets', 'the-events-calendar' ),
+			'action'       => 'activate',
+		];
+
+		$html = $this->get_template()->template(
+			'notices/install-event-tickets',
+			$this->get_template_data( $args ),
+			false
+		);
+
+		tribe_notice(
+			'event-tickets-activate',
+			$html,
+			[
+				'dismiss' => true,
+				'type'    => 'warning',
+			]
+		);
+	}
+
+	/**
+	 * Data for the notice template.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $args Array with arguments to override the defaults.
+	 *
+	 * @return array The template args.
+	 */
+	private function get_template_data( $args = [] ): array {
+		$admin_url    = is_network_admin() ? network_admin_url( 'settings.php' ) : admin_url( 'admin.php' );
+		$redirect_url = add_query_arg( [ 'page' => 'tec-tickets-settings' ], $admin_url );
+
+		$defaults = [
+			'plugin_slug'      => self::$plugin_slug,
+			'action'           => 'install',
+			'title'            => __( 'Start selling tickets to your Events', 'the-events-calendar' ),
+			'description'      => __( 'Get Event Tickets to manage attendee registration and ticket sales to your events, for free.', 'the-events-calendar' ),
+			'button_label'     => __( 'Install Event Tickets', 'the-events-calendar' ),
+			'tickets_logo'     => Tribe__Main::instance()->plugin_url . '/src/resources/images/tec-tickets-logo.svg',
+			'ajax_nonce'       => wp_create_nonce( 'tribe_events_admin_notice_install' ),
+			'redirect_url'     => $redirect_url,
+			'installing_label' => __( 'Installing...', 'the-events-calendar' ),
+			'installed_label'  => __( 'Installed!', 'the-events-calendar' ),
+			'activating_label' => __( 'Activating...', 'the-events-calendar' ),
+			'activated_label'  => __( 'Activated!', 'the-events-calendar' ),
+
+		];
+
+		return wp_parse_args( $args, $defaults );
+	}
+
+	/**
+	 * Handle AJAX response for the notice actions.
+	 *
+	 * @return void
+	 */
+	public function ajax_handle_notice_install_event_tickets() {
+		if ( ! check_ajax_referer( 'tribe_events_admin_notice_install', 'nonce', false ) ) {
+			$response['message'] = wpautop( __( 'Insecure request.', 'the-events-calendar' ) );
+
+			wp_send_json_error( $response );
+		}
+
+		if ( ! current_user_can( 'install_plugins' ) ) {
```

Comment:
Maybe make this a condition of showing the notice in the first place? Do we want to advertise to folks who _can't_ install?

Reply:
It's already in the hooks method. As it's a public method I feel it'd be more robust to have the check here as well.